README.md

```diff
@@ -7,69 +7,190 @@ An [Aerospike](https://www.aerospike.com/) client library for PHP 8+.
 ## Requirements
 
 * PHP 8.1+
-* Composer
 * Cargo
 * Aerospike server 6.3
-* Linux or Darwin (Windows will be added in a future release)
+* Linux or Darwin 
+* PHPUnit v10
+* Rustc v1.74 =<
+* Go Toolchain [Go Toolchains - The Go Programming Language](https://go.dev/doc/toolchain)
+* Protobuf Compiler [protoc-gen-go command - google.golang.org/protobuf/cmd/protoc-gen-go - Go Packages](https://pkg.go.dev/google.golang.org/protobuf/cmd/protoc-gen-go)
+* ext-php-rs v0.12.0 [github repository link](https://github.com/davidcole1340/ext-php-rs/tree/master)
 
 ## Current Limitations
 
-* Does not support Scan/Query API features for Aerospike server 6.4.
+* Does not support Scan/Query API features/
 * Use Aerospike Server 6.3 for testing, the Rust client does not fully support newer versions.
+* Should have the local go daemon setup.
 
```

Comment:
Do you also need to mention that CDTs are not supported?

Reply:
sure, I could add that as well.